kmd.tree

```diff
@@ -22,9 +22,12 @@
         <toc-element id="organize-process-around-kmm.md"/>
         <toc-element id="architect-kmm-app.md"/>
         <toc-element id="connect-to-platform-specific-apis.md"/>
-        <toc-element id="configure-sqldelight-for-data-storage.md"/>
-        <toc-element id="use-ktor-for-networking.md"/>
         <toc-element id="add-dependencies.md"/>
```

Comment:
It seemed to me that it's better to change the order of pages, because adding dependencies is needed in all the three following pages. But surely we can discuss.